plugin/sampling/strategystore/static/strategy.go

```diff
@@ -34,8 +34,14 @@ type serviceStrategy struct {
 	strategy
 }
 
+// defaultStrategy defines the default strategy.
+type defaultStrategy struct {
```

Comment:
I think we can simply use `serviceStrategy`.  The `Service` field can be ignored. Then it will match the structure of the store:
```go
type strategyStore struct {
	logger *zap.Logger

	defaultStrategy   *sampling.SamplingStrategyResponse
	serviceStrategies map[string]*sampling.SamplingStrategyResponse
}
```